src/technogymmyruntreadmillrfcomm.cpp

```diff
@@ -103,6 +103,9 @@ void technogymmyruntreadmillrfcomm::update() {
 
     if (initDone) {
         // ******************************************* virtual treadmill init *************************************
+
+        // TODO: determine why no checking if this has been done before
+
         if (!virtualTreadMill) {
```

Comment:
@cagnulein Why no check if the virtual device setup or "first state changed" here?

Reply:
there is only one use that is using this, so it's a working in progress module